src/components/container/DailyDataChart/DailyDataChart.tsx

```diff
@@ -60,21 +36,25 @@ export default function DailyDataChart(props: DailyDataChartProps) {
     let [currentData, setCurrentData] = useState<DailyDataQueryResult | null>(null);
     let [hasAnyData, setHasAnyData] = useState(false);
 
-    let layoutContext = useContext(LayoutContext);
-
     const dateRangeContext = useContext<DateRangeContext | null>(DateRangeContext);
     let intervalType = props.intervalType || "Month";
     let intervalStart: Date;
 
     if (dateRangeContext) {
-        intervalType = dateRangeContext.intervalType === "Day" ? "Week" : dateRangeContext.intervalType;
+        if(dateRangeContext.intervalType === "SixMonth") {
+            intervalType = "Week"
```

Comment:
Yeah, I was trying to match the pre-existing condition.  However, I think changing it to Month but not changing the condition for Day makes the most sense.  If you "undershoot" and ask for Day, we bump it up to Week (the next smallest thing supported), and if you "overshoot" and ask for 6 Months of data, we go to the largest thing - Month.  I think it's going to be mildly confusing for an end user regardless (because we don't currently do a good job of indicating in Docs what's supported), but in this case they get the *closest* thing to what they might have actually asked for.